python/lsst/ap/association/diaPipe.py

```diff
@@ -637,6 +637,8 @@ def associateDiaSources(self, diaSourceTable, solarSystemObjectTable, diffIm, di
                 toAssociate.append(ssoAssocResult.ssoAssocDiaSources)
             nTotalSsObjects = ssoAssocResult.nTotalSsObjects
             nAssociatedSsObjects = ssoAssocResult.nAssociatedSsObjects
+            self.metadata['nExpectedSsObjects'] = nTotalSsObjects
+            self.metadata['nAssociatedSsObjects'] = nAssociatedSsObjects
```

Comment:
These are both already added to the task metadata in `_add_association_meta_data` below.